deploy-board/deploy_board/templates/environs/env_landing.html

```diff
@@ -345,6 +345,12 @@ <h4 class="panel-title pull-left pointer-cursor">
                 {% endif %}
                 </a>
             </h4>
+            {% if project_info %}
+            <div class="btn-project-info">
+                <span class="glyphicon glyphicon-info-sign" aria-hidden="true"></span>
+                <a href="{{ project_info.project_url }}" target="_blank">Project: {{ project_info.project_name }}</a>
+            </div>
+            {% endif %}
```

Comment:
can you display a warning something like "warning: no project ownership info, please contact slack #infra-console" in the case of no project. 
and group this block into IS_PINTEREST flag.

Reply:
added the warning message when a project does not exist.